DATA/production/qc-workflow.sh

```diff
@@ -32,7 +32,7 @@ if [[ -z $QC_JSON_FROM_OUTSIDE ]]; then
     [[ -z "$QC_JSON_FV0" ]] && QC_JSON_FV0=$O2DPG_ROOT/DATA/production/qc-sync/fv0.json
     [[ -z "$QC_JSON_EMC" ]] && QC_JSON_EMC=$O2DPG_ROOT/DATA/production/qc-sync/emc.json
     [[ -z "$QC_JSON_MCH" ]] && QC_JSON_MCH=$O2DPG_ROOT/DATA/production/qc-sync/mch.json
-    [[ -z "$QC_JSON_MID" ]] && QC_JSON_MID=$O2DPG_ROOT/DATA/production/qc-sync/mid.json
+    [[ -z "$QC_JSON_MID" ]] && has_processing_step MID_RECO && QC_JSON_MID=$O2DPG_ROOT/DATA/production/qc-sync/mid.json || QC_JSON_MID=$O2DPG_ROOT/DATA/production/qc-sync/mid-digits.json
```

Comment:
I am not sure if this `&&` and `||` logic is correct. Don't we need 
```
[[ -z "$QC_JSON_MID" ]] && ( has_processing... && a || b)
```

Reply:
In my tests this logic was working as expected. Anyways, I can have a deeper look next week

Reply:
Hi @davidrohr : indeed, without the parenthesis, the syntax was not working as expected when QC_MID_JSON was already set (it re-defined it anyways).
On the other hand, adding a parenthesis does not seem to work very well on Mac. I therefore modified the syntax with a more standard conditional assignment.